src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
```

Comment:
Instead of using literals in expressions (hardcoded values), define constants with descriptive names to explain these values. For example, use constants like `LONG_TERM_RENTAL_DAYS` and `LONG_TERM_DISCOUNT` to make the code more readable and maintainable. Refer to checklist item 2.